PageObject/Locators.py

```diff
@@ -1,10 +1,12 @@
-class LTLocator(object):
+class LTLocatorHomePage(object):
 
     # Locators for HomePage
     login_link = "//a[normalize-space()='Login']"
     signup_link = "//a[contains(text(), 'Sign Up')]"
     logo = "//a[@href='https://www.lambdatest.com/']/img"
 
+
+class LTLocatorLoginPage(object):
```

Comment:
chyba od pythona 3.0 nie trzeba dziedziczyć po object, to jest robione by deafult czyli powinno wystarczyć:

```suggestion
class LTLocatorLoginPage():
```